core/src/main/scala/spinal/core/formal/Misc.scala

```diff
@@ -9,7 +9,28 @@ object FormalDut{
       c.withAutoPull()
       c.setFormalTester()
     }
+
+    dut match {
```

Comment:
So do you want to run the formal asserts automatically?
IMO explicit is better than implicit.

Reply:
Sort of? I was thinking if the user was calling `FormalDut` on something; that was an explicit action. Otherwise you end up with a lot of little boilerplate code around it. Tastes vary around this kind of thing though; I'll take it out if you want